docs/concepts/getting-started-guide.md

```diff
@@ -3,45 +3,67 @@ id: getting-started-guide
-1. Understand the [Contribution](../contributing/contributing) process and the different tiers and support levels (partner vs community support, etc.).
+1. If you plan to publish your content to the [XSOAR Marketplace](../partners/marketplace) for other customers to use, read about the [Contribution](../contributing/contributing) process and the different tiers and support levels (partner vs community support, etc.).
 1. Bookmark the links to the [Cortex XSOAR Developer Hub](https://xsoar.pan.dev/docs/) (this site) and the [Cortex XSOAR Product Documentation Page](https://docs.paloaltonetworks.com/cortex/cortex-xsoar.html).
 1. Access the Palo Alto Networks [DFIR Slack Community](https://start.paloaltonetworks.com/join-our-slack-community) and join the *#demisto-developers* channel.
 1. Sign up to the [Developer Newsletter](https://start.paloaltonetworks.com/cortex-xsoar-developer-newsletter.html) to receive technical updates on developing and contributing.
 1. Obtain and install a copy of Cortex XSOAR. If you are not a Partner, you can obtain the Community Edition [here](https://start.paloaltonetworks.com/sign-up-for-demisto-free-edition). Installation instructions are available [here](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/installation.html).
 
 ### Other prerequisites (for Integrations and Automations)
 
-If you want to develop more than just Playbooks and you are looking to create Integrations and Automations, you will also need:
+If you want to develop Integrations and Automations, some coding is required. You will need:
 1. Python (3.7 and above) or Powershell programming experience.
-2. (for Integrations) API or SDK access to your product or solution you want to integrate with.
+1. (for Integrations) API or SDK access to your product or solution you want to integrate with.
 
 ### Technology Partners
 
-If you are a Technology Partner, make sure that you also:
+If you are or want to become a Technology Partner, make sure that you also:
 
 1. Read the [Become a Technology Partner](../partners/become-a-tech-partner) page and follow the steps to sign up and sign the agreements.
-2. Work with the [Cortex XSOAR Alliances Team](mailto:soar.alliances@paloaltonetworks.com) to make sure your use cases have been validated.
+1. Work with the [Cortex XSOAR Alliances Team](mailto:soar.alliances@paloaltonetworks.com) to make sure your use cases have been validated.
+
+## Are you planning to contribute?
+
+This site describes how to create content meant to be published on the XSOAR Marketplace and used by several customers. You must following proper design, development and documentation guidelines to make sure that the content can used in production in large SOCs. While it's always recommended to follow all these guidelines,  some steps, such as the dev environment setup, are mandatory only if you are planning to contribute your content.
```

Comment:
```suggestion
This site will guide and offer best practices while creating content to be used in the XSOAR platform. For those of you who want to take their content to the next level such that it will be published in the XSOAR marketplace and used by numerous users worldwide, we offer a full [contribution](../contributing/contributing) guide. We will walk you through proper design, development, and documentation guidelines to enable your content to be used in production environments in large SOCs. While it's always recommended to follow all these guidelines, some steps, such as the dev environment setup, are relevant only if you are planning to contribute your content.
```

Reply:
done